flocx_market/db/sqlalchemy/models.py

```diff
@@ -45,7 +46,16 @@ class Bid(Base):
     @orm.validates('cost')
     def validate_cost(self, key, value):
         if value < 0:
-            raise ValueError('Cost must be >= 0')
+            raise e.InvalidInput('Cost must be >= 0')
```

Comment:
What's the motivation for using `InvalidInput` here rather than the built-in `ValueError` (which seems appropriate for this particular use case)?

Reply:
I did this because the
return json.dumps(e.message), e.code
that every function in the flask api returns requires a message and code, which ValueError does not have. Alternatively I could add an except ValueError part to all the api pieces and return the error code there, if you prefer that solution